### PR TITLE
Add dnt support in s2s

### PR DESCRIPTION
**Pull request by shiralal**
Added support for DO NOT TRANSLATE in client side code.

### CODE REVIEW

File: riva/clients/nmt/riva_nmt_streaming_s2s_client.cc
Comment (rmittal-github) [inline riva/clients/nmt/riva_nmt_streaming_s2s_client.cc]: can we name is `dnt_words_file` since we expect user to provide words instead of phrases

**Comment by rmittal-github** [comment]
@shiralal please rebase on top of release/2.17.0 branch, since it is targeted for upcoming release.

File: riva/clients/nmt/riva_nmt_streaming_s2s_client.cc
Comment (shiralal) [reply riva/clients/nmt/riva_nmt_streaming_s2s_client.cc]: > can we name is `dnt_words_file` since we expect user to provide words instead of phrases

its done

File: riva/clients/nmt/riva_nmt_streaming_s2s_client.cc
Comment (shiralal) [inline riva/clients/nmt/riva_nmt_streaming_s2s_client.cc:97]: Changes added by  `pre-commit run --all-files`

**Review by rmittal-github — APPROVED** [review]
LGTM, will wait for CI to complete before merge

File: WORKSPACE
Comment (rmittal-github) [inline WORKSPACE]: let's update latest common repo commit here

**Pull request merged**
